prime-router/metadata/file_name_templates/file-name-templates.yml

```diff
@@ -71,4 +71,34 @@
   lowerCase: true
   elements:
     - "prime-"
-    - createdDate()
\ No newline at end of file
+    - createdDate()
+
+- name: chester_hl7
+  lowerCase: true
+  elements:
+    - cdcprime_chester_
+    - createdDate()
+
+- name: bucks_hl7
+  lowerCase: true
+  elements:
+    - cdc_prime_bucks_
+    - createdDate()
+
+- name: allegheny_hl7
+  lowerCase: true
+  elements:
+    - cdc_prime_allegheny_
+    - createdDate()
+
+- name: montgomery_hl7
+  lowerCase: true
+  elements:
+    - cdc_prime_montco_
+    - createdDate()
+
+- name: philadelphia_hl7
+  lowerCase: true
+  elements:
+    - cdc_prime_phila_
+    - createdDate()
```

Comment:
Shiny new name formats! So shiny. Do we deserve this shiny? No we do not.

Reply:
Jumping on the bandwagon here - is there a reason we cannot include uuid in the filename to prevent potential filename conflicts?